app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -32,12 +34,12 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
         val presenter = ApplicationPresenter()
         presenter.onViewTaken(this)
 
-        val button = findViewById<Button>(R.id.button)
-        button.setOnClickListener {
+        val searchButton = findViewById<Button>(R.id.button)
+        searchButton.setOnClickListener {
             showLoadingSpinner(true)
             presenter.setNumAdults(numAdults)
             presenter.setNumChildren(numChildren)
-            if (departureTime=="Unset"){
+            if (departureTime=="Today, Now"){
```

Comment:
"Today, Now" seems to be turning into a bit of a magic string, do you think there's maybe a better way to handle the case that the user doesn't manually specify a time?

Reply:
This has been changed now and implemented using a new getTime function as you mentioned in the comment below